src/app/SharpRaven/Data/JsonPacket.cs

```diff
@@ -396,5 +397,10 @@ private void Initialize(Exception exception)
                 }
             }
         }
+
+        public override int GetHashCode()
+        {
+            return this.ToString().GetHashCode()*17;
```

Comment:
Why is this override necessary, and if it is, can't it just return `ToString().GethashCode()` directly? Why multiply by 17?